controllers/validationController.js

```diff
@@ -25,3 +28,25 @@ exports.validateNameAndEmail = [
     next();
     },
 ];
+
+exports.validateAdminPassword = (req, res, next) => {
+    if (process.env.NODE_ENV === 'production') {
```

Comment:
There is one important thing to do here.
The reason why we're using such `NODE_ENV` or `node-config` module is to not use `if` phrase. Meaning, when we change some environment variables, we don't have to come to the code but just change in env file or Heroku config.

So this if should be removed and just remained as 
```
adminPassword = config.get('Admin.adminConfig.password')
```

And then we have a problem. `Admin.adminConfig.password` is `node-config`'s variable, not Heroku. But `node-config` has ability to map its variable with outside `NODE_ENV`. The "outside NODE_ENV" I'm talking here is Heroku's Config var. 

This is explained here https://github.com/lorenwest/node-config/wiki/Environment-Variables#custom-environment-variables

Like in the explanation, we need to map `config.get('Admin.adminConfig.password')` with `ADMIN_PASSWORD` from Heroku.

Reply:
Do we have to modify here in db.js as well? 

![Screenshot 2021-10-07 at 19 35 02](https://user-images.githubusercontent.com/60405643/136435193-a4634fb6-12af-4483-a637-8118797d8690.png)

Reply:
It’s very good point. I think it’s not easy to remove `if` phrase there because one uses id/pwd and other one doesn’t. 
I think we can maybe put `MONGODB_USERNAME` and `MONGODB_PASSWORD` in custom-environment-variables.json. Because it will help developers to understand that there are external variables from outside (Heroku).